docs/content/concepts/spaces-and-transforms.md

```diff
@@ -84,19 +89,19 @@ Say you have a 3D world with two cameras with known extrinsics (pose) and intrin
 
 ```py
 # Log some data to the 3D world:
-rr.log_points("world/points", …)
+rr.log_points("world/points", ...)
```

Comment:
Revert this change here and below

Reply:
Done. Including in the other examples which had `...` before (I opted for `...` because it's [valid Python](https://docs.python.org/3/library/constants.html#Ellipsis) while `…` is not).